packages/trader/src/Modules/CFD/Helpers/constants.js

```diff
@@ -12,6 +12,17 @@ const getTopUpConfig = () => {
     };
 };
 
+const getPlatformDXTradeDownloadLink = (platform = undefined) => {
+    switch (platform || OSDetect()) {
+        case 'ios':
+            return 'https://apps.apple.com/tt/app/dxtrade/id1510582738';
+        case 'android':
+            return 'https://play.google.com/store/apps/details?id=com.devexperts.dxmarket.client&hl=en';
```

Comment:
IMO, it is a good pattern to import these constant URLs from a separate config file that holds these constants instead of hardcoding them directly into code. by the way, it is not a big deal.

```suggestion
            return IOS_APP_URL;
        case 'android':
            return ANDROID_APP_URL;
```